Jenkinsfile

```diff
@@ -284,6 +284,11 @@ pipeline {
     CHANGE_BRANCH = "${env.CHANGE_BRANCH != null ? env.CHANGE_BRANCH : BRANCH_NAME}"
     CHANGE_TARGET = "${env.CHANGE_TARGET != null ? env.CHANGE_TARGET : BRANCH_NAME}"
     CONNECT_PREPROD_URL = 'https://nos-preprod-connect.nuxeocloud.com/nuxeo'
+    ORG = "nuxeo"
```

Comment:
unused

Reply:
This is mandatory

Reply:
why? where?

Reply:
https://jenkins.platform.dev.nuxeo.com/blue/organizations/jenkins/nuxeo%2Fnuxeo/detail/PR-3856/84/pipeline/

in the last shell script before last line:
```
[2020-03-27T20:56:08.654Z] error: no ORG environment variable found

script returned exit code 1
```
I didn't search extensively about why exactly but it is also present in the arender Jenkinsfile

Reply:
I think it's important to understand why it is needed, because reading the Jenkinsfile, defining an environment variable without any obvious reference will lose the reader.
Then, add a comment to explain why/where it is used.

Reply:
It's indeed present in the ARender Jenkinsfile because it is actually used: https://github.com/nuxeo/nuxeo-arender-connector/blob/10/Jenkinsfile#L48.

Reply:
This is a jx preview requirement as well as APP_NAME:
https://github.com/jenkins-x/jx/blob/master/pkg/cmd/preview/preview.go#L1008-L1022